Documentation/observability/metrics.rst

```diff
@@ -530,6 +539,29 @@ and use the IP address of the target instead.
 
    In all of these 3 cases, ``reserved-identity`` context returns ``reserved:kube-apiserver``.
 
+Hubble metrics can also be configured with a ``labelsContext`` which allows providing a list of labels
+that should be added to the metric. Unlike ``sourceContext`` and ``destinationContext``, instead
+of different values being put into the same metric label, the ``labelsContext`` puts them into different label values.
+
+========================= ===============================================================================
+Option Value              Description
+========================= ===============================================================================
+``source_namespace``      The namespace of the pod if the flow source is from a Kubernetes pod.
+``source_pod``            The pod name of the pod if the flow source is from a Kubernetes pod.
+``source_workload``       The name of the source pod's workload (Deployment, Statefulset, Daemonset, ReplicationController, CronJob, Job, DeploymentConfig (Openshift)).
```

Comment:
```suggestion
``source_workload``       The name of the source pod's workload (Deployment, Statefulset, Daemonset, ReplicationController, CronJob, Job, DeploymentConfig (OpenShift)).
```

Here and below. And then I _think_ you can remove Openshift from the spelling_wordlist.txt, because it appears elsewhere spelled as “OpenShift” in the docs without triggering spelling errors.

Reply:
Fixed